Jenkinsfile

```
@@ -61,7 +61,59 @@ spec:
       requests:
         ephemeral-storage: "4Gi"
       limits:
-        ephemeral-storage: "8Gi"            
+        ephemeral-storage: "8Gi"
+
+  - name: hadolint
+    image: hadolint/hadolint:latest-debian@sha256:15016b18964c5e623bd2677661a0be3c00ffa85ef3129b11acf814000872861e
+    imagePullPolicy: IfNotPresent
+    command:
+        - cat
+    tty: true
+    resources:
+      requests:
+        ephemeral-storage: "1Gi"
+      limits:
+        ephemeral-storage: "2Gi"
+
+  - name: dind-daemon
+    image: docker:1.12.6-dind
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 20m
+        memory: 512Mi
+    securityContext:
+      privileged: true
+    volumeMounts:
+      - name: docker-graph-storage
+        mountPath: /var/lib/docker
+    resources:
+      requests:
+        ephemeral-storage: "1Gi"
+      limits:
+        ephemeral-storage: "2Gi"
+
+  - name: maven
+    image: 779921734503.dkr.ecr.eu-west-1.amazonaws.com/jnlp-dood-new-infra:200608
```

So it is the same as the services one and using the newer helm version
```suggestion
    image: 779921734503.dkr.ecr.eu-west-1.amazonaws.com/jnlp-dood-new-infra:200710
```